cmd/fluxctl/list_images_cmd.go

```diff
@@ -39,6 +40,7 @@ func (opts *imageListOpts) Command() *cobra.Command {
 	cmd.Flags().StringVarP(&opts.namespace, "namespace", "n", "", "Namespace")
 	cmd.Flags().StringVarP(&opts.workload, "workload", "w", "", "Show images for this workload")
 	cmd.Flags().IntVarP(&opts.limit, "limit", "l", 10, "Number of images to show (0 for all)")
+	cmd.Flags().BoolVar(&opts.noHeaders, "no-headers", false, "Don't print headers (default print headers)")
```

Comment:
No strong opinions, but `--quiet` and `-q` may be a better name. @hiddeco thoughts?

Reply:
`kubectl get` offers `--no-headers`, while `helm list` uses `-q` (and `--short`).

I think following `kubectl` here is more appropriate and expected. If we would like to settle somewhere in the middle we could use `--no-headers` with `-q` as a shorthand, but it may be wiser to not assign a shorthand at all so that it is available for future usage as a global flag.

Reply:
sounds good to me